docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md

```diff
@@ -205,7 +174,7 @@ print(checkpoint)
 Once we have created the <TechnicalTag tag="checkpoint" text="Checkpoint"/>, we will run it and get back the results from our <TechnicalTag tag="validation" text="Validation"/>.
 
 ```python title="Jupyter Notebook"
-context.run_checkpoint(ge_cloud_id=checkpoint.ge_cloud_id, batch_request=batch_request)
+checkpoint.run()
```

Comment:
as it became a oneliner, should we have a separate step for it?
What do you think about merging it with 4.1?